### PR TITLE
initial test effort

### PR DESCRIPTION
**Pull request by kalantar**

### CODE REVIEW

**Comment by kalantar** [comment]
I was able to see the same problem locally after I did `minikube stop` suggesting the problem is about the kube config

**Review by sriumcp — APPROVED** [review]
The new test is passing. LGTM.

File: .github/workflows/build-and-push.yaml [hunk 1]
```diff
@@ -40,6 +40,12 @@ jobs:
         # (you'll need to set the KUBEBUILDER_ASSETS env var if you put it somewhere else)
         sudo mv /tmp/kubebuilder_2.3.1_${os}_${arch} /usr/local/kubebuilder
         export PATH=$PATH:/usr/local/kubebuilder/bin
+    - name: Set up minikube
```
[the remark this answers is not in the document]
Comment (kalantar) [reply .github/workflows/build-and-push.yaml:43]: I took it out of build and push. I put it in the wrong place initially

**Comment by kalantar** [comment]
I've merged but really only because of the fix in the code. Still not entirely happy with this approach to testing.

**Pull request merged**